src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -3,7 +3,10 @@
 ## Unreleased
 
 * LogExporter bug fix to handle null EventName.
-  ([#2870](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2871))
+  ([#2871](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2871))
+
+* Fixed the default endpoint for OTLP exporter over HTTP/Protobuf.
```

Comment:
nit: if we can mention the new default endpoint right here, it'll save some time for users as they dont have to open PR/issue.

Reply:
Fixed: #a2b6c07530f8246eca87d30f981247b3d98af193